contents/handbook/strategy/overview.md

```diff
@@ -41,30 +41,37 @@ We now add more companies each week than _any_ closed source SaaS rival, and we'
 
 **From where we are today, what do we need to build to get there?**
 
-* PostHog becomes a data platform, with a wider range of data ingestion. We make warehouses redundant.
-  * Today, teams often export everything to a warehouse and query it there. Then they end up with a data engineering mess trying to set up other tools with their warehouse and / or a data platform.
+* PostHog becomes a data platform, with a wider range of data ingestion (both real time streaming, and in bulk so migration is possible). We make warehouses redundant as a result.
+  * Today, teams often export everything to a warehouse and query it or send it to other products from there. Then they end up with a data engineering mess trying to set up other tools with their warehouse and / or a data platform. We already have lots of exports, but lack imports. This would help data engineers service the rest of their organization better.
 * We need to provide a more flexible UX, including SQL, so users can measure and diagnose product performance better through direct data access.
   * To remove the need for a warehouse, we must give direct access to data.
 * Before other tools are built into PostHog, we make it easy to get more value out of them, with PostHog's data. Over time, we will build these all into the platform.
-  * We can enrich other products through our browser extension and export apps. Over time, we will remove the need for these 3rd party products.
+  * We can enrich other products through our browser extension and export apps. Over time, we will remove the need for these 3rd party products. In the short run, these two methods will enable data engineers to help other teams start using PostHog data (marketing/customer success/sales), before we have built front end apps for in the immediate future.
 * We need to make sure our existing products in the platform have any major gaps in functionality closed, and that they integrate well with the rest of the product.
   * This allows us to move people from 3rd party products, for a simpler stack and faster growth and reinvestment.
-* We need to make users _love_ the platform rather than _like_ it, through a user experience that makes developers feel at home.
-  * This will increase word of mouth growth. We shouldn't just build an open source alternative of competitors' products, which are more focused on less technical people.
-
+* We need users to _love_ the platform rather than _like_ it, through a user experience that makes developers feel at home.
+  * This will increase word of mouth growth. We shouldn't just build an open source alternative of competitors' products, which have a UX and UI focused on less technical people.
```

Comment:
Could potentially be more precise here for clarity - 'which have a UX and UI focused on non-technical product and marketing people, rather than engineers.'

(The reason I think this is helpful is it gives an obvious answer to questions like 'should we prioritize this marketing use case or data engineering use case.')

Reply:
reworded version that covers this in more depth, incoming in #4670